Fix three-three-fiber peer dependency typo

AustinBrunkhorst opened the pull request:
It appears `three-three-fiber` was erroneously added as a peer dependency, which produces a warning on `yarn install`.

drcmda commented:
it does depend on it though? why would it cause a warning?

AustinBrunkhorst commented:
> it does depend on it though? why would it cause a warning? https://unpkg.com/browse/drei@0.0.32/dist/index.js

It does depend on `react-three-fiber`; however, the peer dependency declares `three-three-fiber` which appears to be a typo.

drcmda commented:
Oh, didn't see it like that, could you fix the typo instead? In theory react-three-fiber should be a peer dep

AustinBrunkhorst commented:
Moved it to peer dependencies. I also updated `react` and `react-dom` as it looks like they were declared in dev dependencies.

Should we also move `three` to peer dependencies? This seems like a good candidate.

drcmda commented:
i think its fine, r3f has a peerdep on three, it should warn if three is not there. i noticed that TS doesnt compile if react and react-dom is removed from devdeps. i have just fixed the typo and published it. thanks! :-)
